consensus/istanbul/backend/api.go

```diff
@@ -185,7 +185,7 @@ func (api *API) GetValEnodeTable() (map[string]*vet.ValEnodeEntryInfo, error) {
 }
 
 func (api *API) GetVersionCertificateTableInfo() (map[string]*vet.VersionCertificateEntryInfo, error) {
-	return api.istanbul.versionCertificateTable.Info()
+	return api.istanbul.announceManager.versionCertificateTable.Info()
```

Comment:
i would hide `versionCertificateTable` and instead expose a getter for the `Info` on the announce manager

Reply:
I'll change it